fenix/docs/Guide-to-merging-contributor-PRs.md

```diff
@@ -1,52 +1,22 @@
 # Guide to merging contributor PRs for Firefox Android team members
 
-Contributor PRs will run only a specific suite of CI tests (excluding UI tests) in order to protect secrets. Once a Firefox Android team member has reviewed the PR and deemed it safe, we can use the following steps to run the full CI suite and land the patch.
+Contributor PRs will run only a specific suite of CI tests (excluding UI tests) in order to protect secrets. Use the following steps when reviewing and merging a contributor PR. 
 
-## Command line instructions
-
-_Note: these instructions use https://cli.github.com/_
-
-1. Fetch upstream changes and locally check out the contributor's branch onto your fork
+1. Fetch upstream changes and locally check out the contributor's branch onto your fork.
 ```sh
 git fetch --all
 gh pr checkout <PR number>` or `gh pr checkout <PR number> -R mozilla-mobile/fenix
```

Comment:
Perhaps we could just simplify this to be just 
gh pr checkout <PR number>

I have only ran the command (without the fetch) that I see from the following:
<img width="440" alt="Screen Shot 2023-05-18 at 1 04 39 PM" src="https://github.com/mozilla-mobile/firefox-android/assets/1190888/bdd280dc-ce92-4b9b-90c6-8202d09f1507">

Reply:
Sure, I'll update. thanks